.travis.yml

```diff
@@ -12,6 +12,12 @@ matrix:
       cache: false
     - sudo: 9000
       dist: trusty
+      before_install:
+        - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+        - sudo add-apt-repository 'deb http://apt.llvm.org/precise/ llvm-toolchain-precise-3.9 main' -y
```

Comment:
This should likely be `trusty` instead of `precise`, since we have `dist: trusty` in the same file.